src/main/java/crazypants/enderio/gui/TooltipAddera.java

```diff
@@ -286,6 +286,20 @@ public static void addTooltipFromResources(List list, String unlocalizedName, St
     }
   }
 
+  public static void addCommonTooltipFromResources(List list, ItemStack itemstack) {
+    if(itemstack.getItem() == null) {
+      return;
+    }
+    String unloc = null;
+    if (itemstack.getItem() instanceof IResourceTooltipProvider) {
```

Comment:
This is now duplicated code from below. Should extract this into a private helper method that is called from multiple places.